src/tlo/methods/wasting.py

```diff
@@ -1,41 +1,1442 @@
+                        'MUAC<115mm alone'),
+        'rr_SAM_death_kwashiorkor_both_WHZ<-3_&_MUAC<115mm': Parameter(
+            Types.REAL, 'relative risk of death from SAM if bilateral oedema present, WHZ<-3 and MUAC<115mm, '
+                        'compared to MUAC<115mm alone'),
+        # treatment parameters
+        'coverage_supplementary_feeding_program': Parameter(
+            Types.REAL, 'coverage of supplementary feeding program for MAM in health centres'),
+        'coverage_outpatient_therapeutic_care': Parameter(
+            Types.REAL, 'coverage of outpatient therapeutic care for SAM in health centres'),
+        'coverage_inpatient_care': Parameter(
+            Types.REAL, 'coverage of inpatient care for complicated SAM in hospitals'),
+        'prob_mam_death_after_care': Parameter(
+            Types.LIST, 'probability of dying or returning to MAM after seeking care'),
+    }
 
     PROPERTIES = {
-        'un_clinical_acute_malnutrition': Property(Types.CATEGORICAL,
-                                                   'temporary property', categories=['MAM', 'SAM', 'well']),
-        'un_ever_wasted': Property(Types.BOOL, 'temporary property')
+        # Properties related to wasting
+        'un_ever_wasted': Property(Types.BOOL, 'had wasting before WHZ <-2'),
+        'un_WHZ_category': Property(Types.CATEGORICAL, 'weight-for-height z-score group',
+                                    categories=['WHZ<-3', '-3<=WHZ<-2', 'WHZ>=-2']),
+        'un_last_wasting_date_of_onset': Property(Types.DATE, 'date of onset of latest wasting episode'),
+
+        # Properties related to clinical acute malnutrition
+        'un_clinical_acute_malnutrition': Property(Types.CATEGORICAL, 'clinical acute  malnutrition state based on WHZ',
+                                                   categories=['MAM', 'SAM'] + ['well']),
+        'un_am_bilateral_oedema': Property(Types.BOOL, 'bilateral oedema present in wasting'),
+        'un_am_MUAC_category': Property(Types.CATEGORICAL, 'MUAC measurement categories',
+                                        categories=['<115mm', '115-<125mm', '>=125mm']),
+        'un_sam_with_complications': Property(Types.BOOL, 'medical complications in SAM'),
+        'un_sam_death_date': Property(Types.DATE, 'death date from severe acute malnutrition'),
+        'un_am_recovery_date': Property(Types.DATE, 'recovery date from acute malnutrition'),
+        'un_am_discharge_date': Property(Types.DATE, 'discharge date from treatment of MAM/ SAM'),
+        'un_acute_malnutrition_tx_start_date': Property(Types.DATE, 'intervention start date'),
+        'un_am_treatment_type': Property(Types.CATEGORICAL, 'treatment types for acute malnutrition',
+                                         categories=['standard_RUTF', 'soy_RUSF', 'CSB++', 'inpatient_care'] + [
+                                             'none', 'not_applicable']),
     }
 
+    wasting_states = ['WHZ<-3', '-3<=WHZ<-2', 'WHZ>=-2']
```

Comment:
@tbhallett . MAM/SAM/well is a diagnosis after looking at things like wasting states( ['WHZ<-3', '-3<=WHZ<-2', 'WHZ>=-2']), Oedama state(True or False) or MUAC states( ['<115mm', '115-<125mm', '>=125mm'] )

Reply:
@mnjowe, But since the categories in property 'un_WHZ_category' are the same as those in 'wasting_states', do we really need to define them again? Can't we just use these categories instead whenever we use 'wasting_states'?